rust/src/editor.rs

```diff
@@ -322,22 +385,22 @@ impl Editor {
                     self.delete_backward();
                 }
                 "\u{F700}" => {  // up arrow
-                    self.move_up(flags);
```

Comment:
With these changes, `flags` becomes a dead variable. But for clients that still send raw keys, the modify-selection behavior is broken. Perhaps it should say, `if flags & MODIFIER_SHIFT { self.modify_selection(); }`